app/controllers/users.py

```diff
@@ -376,20 +377,38 @@ def delete(self, user_id):
         except Exception as e:
             return dict(status='fail', message=str(e)), 500
 
+    @jwt_required
     def patch(self, user_id):
         """
         """
         try:
-            user_schema = UserSchema(only=("name",))
+            user_schema = UserSchema()
```

Comment:
We do this to limit fields that will be updated so when you open in up, it may cause issues long term
```suggestion
      user_schema = UserSchema(only=("name", "public"))
```